db/migrate/20241017114649_add_module_name_to_context.rb

```diff
@@ -0,0 +1,5 @@
+class AddModuleNameToContext < ActiveRecord::Migration[7.0]
+  def change
+    add_column :contexts, :module_name, :string
```

Comment:
For these migrations, are you planning on backfilling the module_name and extractor_name for any current contexts in production? (Right now there is only 1 context, which if I think about it makes sense, since only 1 project has been using this so far 😆 )

Reply:
Yes, plan is to use `galaxy_zoo` and `cosmic_dawn` or in this new case `euclid` for the context `module_name` and `extractor_name` respectively. We have just one context on the db currently so the aim is to update that after migration

Reply:
Please do be sure to document this databack fill ToDo item!